receiver/solacereceiver/config_test.go

```diff
@@ -84,7 +84,7 @@ func TestConfigValidateMissingAuth(t *testing.T) {
 	cfg := createDefaultConfig().(*Config)
 	cfg.Queue = "someQueue"
 	err := component.ValidateConfig(cfg)
-	assert.Equal(t, errMissingAuthDetails, err)
+	assert.ErrorContains(t, err, errMissingAuthDetails.Error())
```

Comment:
Why not "ErrorIs"?

Reply:
The motivation for these changes is https://github.com/open-telemetry/opentelemetry-collector/pull/12108, which prints the path in the config that produced this error. I don't want to depend on the format of the error message produced by `component.ValidateConfig`, only the part of the error produced by this component.

Reply:
I am going to merge the PR to unblock the work on core, since this are tests if you want to change it after the PR is merged we can do that